test-harness/src/test/groovy/com/netflix/conductor/test/integration/SubWorkflowSpec.groovy

```diff
@@ -196,6 +183,15 @@ class SubWorkflowSpec extends AbstractSpecification {
             tasks[1].status == Task.Status.COMPLETED
         }
 
+        and: "subworkflow task is in a completed state"
+        with(workflowExecutionService.getTask(subworkflowTaskId)) {
+            status == Task.Status.COMPLETED
+            subworkflowChanged
+        }
+
+        and: "the parent workflow is swept"
+        sweep(workflowInstanceId)
+
         and: "the parent workflow has been resumed"
         with(workflowExecutionService.getExecutionStatus(workflowInstanceId, true)) {
             status == Workflow.WorkflowStatus.COMPLETED
```

Comment:
Also verify that the subworkflowChanged flag has been reset.

Reply:
Done